driver-scala/src/it/scala/org/mongodb/scala/documentation/DocumentationChangeStreamExampleSpec.scala

```diff
@@ -32,18 +34,15 @@ import org.mongodb.scala.{
   SingleObservable,
   Subscription
 }
-import org.mongodb.scala.model.changestream.ChangeStreamDocument
-import org.mongodb.scala.model.{ Aggregates, Filters, Updates }
 
 import scala.collection.mutable
-import scala.concurrent.Await
-import scala.concurrent.duration.Duration
+import scala.concurrent.{ Await, ExecutionContext }
 
 //scalastyle:off magic.number regex
 class DocumentationChangeStreamExampleSpec extends RequiresMongoDBISpec with FuturesSpec {
 
   "The Scala driver" should "be able to use $changeStreams" in withDatabase { database: MongoDatabase =>
-    assume(serverVersionAtLeast(List(3, 6, 0)) && !hasSingleHost())
+    assume(false) // Don't run in tests
```

Comment:
How come false?

Reply:
I dont want to run these documentation tests as part of the test suite,  just compiling them is fine. This is the same as the java sync equivalent. Except here I don't have an examples folder.